


# Conclusiones



---

## Conclusions of the paper

Larger genetic variability in CMS (\\\(> H_E, > D_G\\\))
CMS5: most diverse accession (\\\(H_E=0.250\\\))
{: .quitesmaller }

QuantStudio: suitable for high-throughput genotyping in sugar beets
Selected SNPs: suited to estimate genetic variability and genetic relationships among sugar beet accessions, and to discriminate between them
{: .quitesmaller }

Wide genetic basis: important in breeding programmes to:
 - select for disease resistance
 - prevent inbreeding depression
 - allow for adaptation to changing environmental conditions
{: .quitesmaller }

---

## Future work

- Minimum set of SNPs for the discrimination of sugar beet accessions;
<!-- - GWAS for disease resistance (CLS, rhizomania ...?) -->
- Domestication of *B. vulgaris* from *B. maritima*

---

## Publications

- Stevanato P., Trebbi D., Broccanello C., Concheri G., Squartini A., Saccomani M., Sablok G. (2012) *High-throughput SNP genotyping in sugar beet*. Proc. International Conference on Bioinformatics and Computational Biology, Sept. 20-21, Varna (Bulgaria)
{: .quitesmaller }
- Stevanato P., Richardson K., Panella L., Sablok G., Broccanello C., Biscarini F., Del Corvo M., Stella A., Saccomani M. (2013) *SNP linked to key root traits in sugar beet.* Proc. of Plant & Animal Genome XXI Conference, San Diego (CA), USA
{: .quitesmaller }
- Stevanato P., Broccanello C., Biscarini F., Del Corvo M., Sablok G., Panella L., Stella A., Concheri G. (submitted) *First application of QuantStudio platform for high-throughput SNP marker assessment in sugar beet* Plant Molecular Biology Reporter 
{: .quitesmaller }

